### DISCUSSION TITLE
Formula for Perfect MB size in HDDs

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#1927] [Show and tell]
Set heads and sectors to 64 and 32, respectively. Then, for cylinders, set that to how many megabytes you want. 1024 cylinders for a gigabyte.

**Comment by ppgrainbow** [comment]
Well...for a more accurate hard drive capacity, I looked at the WinBox Hard Disk Size database and found that the highest capacity hard disk that is under 1 GB is the IBM 0063-H12/L12 with 2051 cylinders, 15 heads and 66 sectors per track for a total of 2,039,490 sectors or 991.45 MB.

The closest value to use for a SCSI hard disk controller, such as the Adaptec AHA-1640A that supports 64 heads and 32 sectors per track would be 991 MB.
